instruqt-tracks/terraform-cloud-aws-v2/track_scripts/setup-workstation

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash -l
+# Track setup script
+
+set -e
+
+# Wait for Instruqt to finish booting the VM
+# This is better than doing sleep
+while [ ! -f /opt/instruqt/bootstrap/host-bootstrap-completed ]
+do
+    echo "Waiting for Instruqt to finish booting the VM"
+    sleep 1
+done
+
+# To repurpose this script, we choose the target 
+# Cloud Environment using this variable
+# export CLOUD_ENV="aws"
```

Comment:
This is used later in this script but I do not see where it is defined, and it is commented out here

Reply:
config.yml

Reply:
Oh, duh, I see it now. Thanks.

Reply:
Maybe we should add a comment as to where that is configured.

Reply:
+1 on that, it will help us in six months when we don't remember the details.